frontend-shared/src/components/test/SvgIcon-test.js

```diff
@@ -15,8 +15,8 @@ describe('SvgIcon', () => {
 
     registerIcons(
       {
-        'arrow-left': require('../../images/icons/arrow-left.svg'),
-        'arrow-right': require('../../images/icons/arrow-right.svg'),
+        'arrow-left': require('../../../images/icons/arrow-left.svg'),
```

Comment:
Can you help me understand this change? Did something move?

Reply:
Ya, I thought we could use the /components folder here too to make it consistent with client